cmd/kube-controller-manager/app/bootstrap.go

```diff
@@ -39,6 +42,7 @@ func startBootstrapSignerController(ctx context.Context, controllerContext Contr
 }
 
 func startTokenCleanerController(ctx context.Context, controllerContext ControllerContext) (controller.Interface, bool, error) {
+	ctx = klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "token-cleaner"))
```

Comment:
maybe we could consider having a function for this, if this is a common pattern?

Reply:
It is common, not just in kube-controller-manager, but also elsewhere. However, the only purpose of a helper function would be to avoid some typing. I prefer to see in the code what's going on instead of having to look up a helper function with a one-line body elsewhere - might just be my personal preference, though.